### ISSUE TITLE
Login function

### ISSUE THREAD

**Issue by jeremyoha450** [rustdesk/rustdesk-server#73]
Hi there

anyway to use the login function of the clients so that if i login it shows the list of hosts i've used/connected to?

like team viewer does? 

regards
Jeremy

**Comment by paspo** [comment]
AFAIK it's not implemented yet.

**Comment by simeononsecurity** [comment]
If this is the case, some idiot proofing would be nice. At least grey the option out until it's ready...
We shouldn't have to ask..

**Comment by paspo** [comment]
Let me rephrase that.
I mean: it's not implemented yet _in the open source release_.

I suppose this feature was working in the previous iteration of RustDesk, when the project was closed-source.

I think @rustdesk just needs time to figure out a way to open source most of this project and still make a living out of it. Personally, I totally understand and respect that.

**Comment by simeononsecurity** [comment]
The easiest and most straight forward is to just charge businesses for continued support and possibly for custom builds or licensing for repackaging.
It's not like we have to redefine the wheel here. Look at what other people do and see what works best for you.
You're not going to attract business with releasing a half-open source product that is half-self-hostable that is missing a majority of needed features other than core functionality and has absolutely no security in mind and minimal documentation.
These are problems that every project faces. They have been solved before. What is ahead is work. Open source the rest of the project so hopefully the community can help along the way and implement a business model for charging enterprise customers for support. 
No sense in fighting it.

**Comment by Gunnar-GO** [comment]
The idea to generate an useable adressbook would be great for private installations. That would make support much more easier and faster.

**Comment by pustekuchen91** [comment]
user authentication and a shared address book would be a teamviewer killer. 

thanks for this nice piece of software

**Comment by bigretromike** [comment]
Related: 
#103 
#109 
#27 
#26

**Comment by henriquelucas** [comment]
Is there any news about configuring this Login issue. I saw that there is an option within Rustdesj, but how to access it?

**Comment by bigretromike** [comment]
> Is there any news about configuring this Login issue. I saw that there is an option within Rustdesj, but how to access it?

You currently don't access it. Its a GUI placeholder from not_public version that had this functionality.

**Comment by henriquelucas** [comment]
Will something else be released in this regard?

**Comment by hassansf** [comment]
Goodbye to Anydesk and Teamviewer.
Thanks to dev.

**Comment by simeononsecurity** [comment]
> Will something else be released in this regard?

They haven't released a client update in over a year. Fingers crossed anything is released any time soon.

**Comment by TopHatProductions115new** [comment]
I just hope it'll have AD/LDAP integration in the future. I'm willing to wait for that.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk-server/issues/158#issuecomment-1545385247
